### PR TITLE
Update CI Go Version to 1.19 (and other formatting nits)

### PR DESCRIPTION
**Pull request by josephlr**
By upgrading our default GI version to Go 1.19, we now catch more lints/nits than we did before. To make the linter happy we:
  - Remove uses of the `ioutil` library
  - Add a package comment for the `cmd/main` package
  - Fix comment formatting in `attest.proto` (which also causes the generated `.pb.go` to change whitespace).
  - Run gofmt over the entire tree (which makes some changes, but only whitespace changes).

This reduces the patches we need when importing into google3

Signed-off-by: Joe Richey <joerichey@google.com>

### CODE REVIEW

File: cmd/gotpm/main.go [hunk 1]
```diff
@@ -1,3 +1,4 @@
+// Package main is a just a binary wrapper package around cmd.
```
Comment (JoshuaKrstic) [inline cmd/gotpm/main.go:1]: nit: Consider removing the word "just"

Comment (josephlr) [reply cmd/gotpm/main.go]: FIxed

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -111,8 +111,7 @@ jobs:
       - name: Run golangci-lint
         uses: golangci/golangci-lint-action@v3.2.0
         with:
-          # Update when https://github.com/golangci/golangci-lint/issues/2997 is fixed.
```
Comment (alexmwu) [inline .github/workflows/ci.yml:114]: does `version: latest` not work anymore? I'd prefer to do that if the regression is fixed than run with an older version

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -125,11 +124,12 @@ jobs:
             --exclude-use-default=false
             --max-same-issues=0
             --max-issues-per-linter=0
```
Comment (alexmwu) [inline .github/workflows/ci.yml:126]: nit: `--timeout 2m`

**Review by alexmwu — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 3 above)
Comment (josephlr) [reply .github/workflows/ci.yml:126]: Fixed

File: .github/workflows/ci.yml (hunk 2 above)
Comment (josephlr) [reply .github/workflows/ci.yml:114]: Fixed, using version: latest seems to work.

**Pull request merged**